website/templates/institution.mako

```diff
@@ -16,12 +16,14 @@
                     <div class="col-sm-3 col-sm-offset-2"><img class="img-circle" height="110px" width="110px" src=${ logo_path }></div>
                     <div class="col-sm-3">
                         <h2>${ name }</h2>
-                        % if description:
-                            <h4><small class="hidden-sm">${description}</small></h4>
-                        % endif
                     </div>
                 % endif
             </div>
+            % if description:
```

Comment:
In my opinion, there is too big a gap in between the banner_path and the description.  And I think the description font is too large? 
![screen shot 2016-04-19 at 6 26 54 pm](https://cloud.githubusercontent.com/assets/9755598/14657520/68ba15dc-065c-11e6-8be4-afd5f411549e.png)

I don't know what discussions have gone on about this but is centering the description below the nice logo the best place to put this?